robot/2023/src/main/java/frc/robot/Robot.java

```diff
@@ -80,13 +94,33 @@ public void teleopInit() {
     if (m_autonomousCommand != null) {
       m_autonomousCommand.cancel();
     }
+
+    stick.setXChannel(LEFT_STICK_VERTICAL);
+    stick.setYChannel(RIGHT_STICK_VERTICAL);
+    stick.setZChannel(RIGHT_TRIGGER);
+    // stick.setZChannel(LEFT_TRIGGER);
+
+    //leftMotorControlOutput.Set(VictorSPXControlMode::PercentOutput, 0.5);
+    //leftMotorControlOutput.set(VictorSPX.PercentOutput, (stick.getRawAxis(2) / 2) - (stick.getRawAxis(3) / 2));
+   /*  public int pct(double raw, int cal) {
+      return (int) (raw * throttle * 1.0);
+    }*/
   }
 
   /** This function is called periodically during operator control. */
   @Override
   public void teleopPeriodic() {
-    System.out.println("Hola allí.");
+
+    double RightTriggerOut = stick.getRawAxis(RIGHT_TRIGGER) * 0.2;
+    double LeftTriggerOut = stick.getRawAxis(LEFT_TRIGGER) * 0.2;
+
+    leftMotorController.set(VictorSPXControlMode.PercentOutput,stick.getRawAxis(LEFT_STICK_VERTICAL));
+    leftMotorController.setInverted(true);
```

Comment:
Good idea!

Reply:
Should this line move from periodic to init?

Reply:
Yes, it has now been moved into robotInit